docs/user/app_use_cases.md

```diff
@@ -62,9 +62,13 @@ Run the following command from the NetBox root directory to create a JSON file (
 ./manage.py dumpdata \
     --traceback \
     --format=json \
+    --indent=2 \
     --natural-primary \
     --natural-foreign \
-    --exclude extras.ObjectChange \
+    --exclude=extras.ObjectChange \
+    --exclude=extras.Report \
+    --exclude=extras.Script \
+    --exclude=django_rq.Queue \
     --output=/tmp/netbox_data.json
```

Comment:
Does the list of `--exclude` options need to differ by NetBox version? Or do the same options apply to all supported versions at this time?

Reply:
Yes it differ. Prior to NetBox 3.7, it was possible to export everything. With NetBox 3.7 I was facing issues dumping mentioned models, so that's why I added these excludes.

Reply:
Ok, so do we need to document different instructions for different NetBox versions?

Reply:
There is the following sentence already:

```
It's possible to exclude other models from the export, but the command provided above is a good starting point.
```

Not sure whether adding multiple examples for multiple NetBox versions wouldn't make it less readable. I would rather keep it as is and let the user figure it out based on their version and preference. WDYT?

Reply:
How should the user figure it out? Do we get clear messages if they exclude the wrong models or fail to exclude the correct models?

Reply:
Reverted `--excludes`. Added an error message example and how to handle that.